docs/pages/reference/operator-resources/resources.teleport.dev_accesslists.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: TeleportAccessList
+description: Provides a comprehensive list of fields in the TeleportAccessList resource available through the Teleport Kubernetes operator.
+tocDepth: 3
+---
+
+{/*Auto-generated file. Do not edit.*/}
+{/*To regenerate, navigate to integrations/operator and run "make crd-docs".*/}
+
+This guide is a comprehensive reference to the fields available to configure in
+the `TeleportAccessList` resource, which you can apply after installing
+the Teleport Kubernetes operator.
+
```

Comment:
That sounds great! What do you think would be the best way to include examples? Map each example file to a CRD in `formatAsDocsPage` based on the example filename?

Reply:
We could have formatAsDocsPage lookup a yaml file named like the CRD and include it if found.

Reply:
Sounds good! Since this is a Q2 goal for us, would it make sense to merge the docs generator this quarter, then work on the examples as time opens up?